app/models/work_package/exporter/formatters.rb

```diff
@@ -1,19 +1,19 @@
 module WorkPackage::Exporter
   module Formatters
+    @@formatters = %i[default costs estimated_hours].map do |key|
```

Comment:
I prefer the memoized style with an instance-level variable. Class variables are generally frowned upon due to the inheritance problems. Even though this module is likely not being subclassed, why introduce a class variable?

Reply:
Yes, I already asked Jens how to improve that because I knew that it "smelled". I'll improve that. Thanks.

Reply:
changed it.